help/C/Help_ch_Transactions.xml

```diff
@@ -1067,28 +1067,112 @@ Author:
     </sect2>
     <sect2 id="trans-import-matcher">
       <title>Import Matcher</title>
-      <para>The Import Matcher uses a Bayesian approach to assign a destination account, if it is not specified in 
+      <para>The Import Matcher uses a Bayesian approach to assign a transfer account, when it is not specified in 
           the imported data, to each imported transaction based on the previous import history of the import account.
-          It also attempts to match the transactions being imported to any existing transactions
-          based on the date and the description fields<!-- FIXME add any other fields used in matching -->.
+          It also attempts to match the transactions being imported to any existing transactions already recorded
+          based on the date, amount and the description fields by tokenizing that information and comparing it to stored tokens for 
+          assigned transfer accounts assigned in previous imports.<!-- FIXME add any other fields used in matching -->.
       </para>
-      <para>Transaction rows which match existing transactions already in the import account are flagged not to be imported. They 
-        will have a light green background and the A and U+R checkboxes will be unchecked and the R checkbox will be 
-        checked. To override and import the transaction, check the A checkbox. The U and R boxes will be unchecked 
-        automatically. The reliability of the match is indicated by a bar display in the Info column. If a destination
-        account for the second split is assigned by the matcher if will be appended to the info column.
-      </para>
-      <para>Transaction rows which do not match existing transactions in the import account, for which an assignment
-        of a destination account cannot be made on the basis of the previous import history to the account, will be 
-        displayed with an orange-yellow background and the A box will be checked and U+R and R unchecked. A destination 
-        account must be specified for these transactions.
+      <para>The status of the imported transaction rows is indicated by a combination of the background colour for the row and three
+       checkboxes in the columns labelled A (Add), U+R (Update and Reconcile) and R (Reconciled) as described in <xref linkend="match_status"/>.
       </para>
+      <note><para>Note that reconciled in this context is not the same as reconciled in the process of reconciling an account.</para></note>
```

Comment:
This needs a better explanation. Isn't it becoming "cleared" in the reconcile dialog?
@jralls should it be changed to from R/Reconciled to C/Cleared in the dialog + updates in other parts of the documentation like screenshots?

Reply:
@fellen, Do you mean should we change the heading in the dialog from U+R to U+C? If that's what it really does, I agree that we should.

Reply:
With the OFX and CSV imports it certainly marks the transactions as "c" in the R column in the register for new records imported which aren't matched. I am not sure what  occurs if an update to an existing transaction which is already reconciled and marked as such by the reconciliation procedure for an account. When the transaction is flagged as R in the matcher it really only implies that the transaction has been matched so maybe an "M" and "U+M" may be more appropriate even that the "C" and "U+C". I would guess that where an import updates an existing transaction which was already reconciled, it should undo the reconciliation status "y" in the register and set it back to "c", particularly if it was the amount which was updated but even if a description was changed since reconciliation really implies that you are satisfied that the transaction in the external statement and the transaction recorded in Gnucash are one and the same, not just that the balances are right at the end date of a reconciliation.

Reply:
@jralls  If you think the above changes make sense and won't be too confusing for long term users, I am happy to make the changes them in the code and then the documentation to match. Would it be better perhaps to raise this as a bug to facilitate any discussion/
David

Reply:
@DaveC49 A(dd) is for new transactions that don't have a match in the book, I don't think they're what we're concerned with here. 
The R checkbox on the importer marks a match as cleared, unchecking it neither adds a transaction nor changes the existing one, so I think that "C" is still correct for that column.
As for whether U+C should unreconcile the split, that seems to be a bit controversial depending on what gets changed. If the amount changes then it's obvious that the split should be unreconciled, but @gjanssens has gotten a lot of flack for his change to the register logic that unreconciles for any change at all to a reconciled split or to its parent transaction.

Changing the headings might be a little confusing to some existing users, but fixing them will be less confusing to new ones. It also changes translatable strings. Perhaps it would be better to put that part in master.

Reply:
@jralls thanks John. I'll create a bug for it and change the U+R to U+C and R to C in the code. If Geert has copped flack for having it change the reconcile status, then that is better left in the user's hands. I'll do that separately from the other issues Frank raised above. Most of these have already been done in a couple of commits I haven't pushed through yet.